src/applications/mhv-landing-page/components/HeaderLayout.jsx

```diff
@@ -78,7 +79,15 @@ const HeaderLayout = ({ showWelcomeMessage = false }) => {
                 <p>
                   If you’re not ready to try the new My HealtheVet, you can use
                   the previous version anytime.{' '}
-                  <a href={goBackUrl} data-testid="mhv-go-back-1">
+                  <a
+                    onClick={() =>
+                      datadogRum.addAction(
+                        'Click on Landing Page: Intro - Go back to the previous version of My HealtheVet',
```

Comment:
Maybe define a const with the link's text and use that here to limit duplication or future changes. This would apply to the other similar calls. No need to have one global variable, just at least one in the file.